workers/data_refinery_workers/processors/multiqc.yml

```diff
@@ -0,0 +1,22 @@
+---  # "MultiQC" processor
+
+# Operating system distribution (such as Ubuntu 16.04, CentOS 7.2, etc)
+os_distribution:
+
+# OS-level packages (installed by OS package manager)
+os_pkg:
+  - python3
+  - python3-pip
+  - default-jre   # required by fastqc
+
+
+# Command lines (assuming "<cmd> --version" is supported)
+cmd_line:
+  - salmon
+  - /home/user/FastQC/fastqc
```

Comment:
I confess that this is a little awkward. I tried to create a soft link in `/usr/local/bin`, but the command complains that it has two many levels of symbolic links.